webclient/js/elems/raflow/rentables.js

```diff
@@ -742,7 +754,15 @@ window.loadRARentablesGrid = function () {
                                 var BID = getCurrentBID();
                                 app.raflow.arList[BID].forEach(function(item) {
                                     if (event.value_new.id == item.ARID) {
+                                        console.log(item);
                                         f.record.Amount = item.DefaultAmount;
+
+                                        if(item.FLAGS === 66){
+                                            f.record.RentCycle = app.cycleFreq[0];
+                                        }
+
+                                        console.log(item.FLAGS);
```

Comment:
remove/comment console.log if it's no longer used

Reply:
As It is in progress. It requires. It'll be removed at end of the feature development completed.

Reply:
all right